### DISCUSSION TITLE
Non Root user for docker images

### DISCUSSION THREAD

**Discussion by rvramesh** [dotnet/dotnet-docker#3208] [Q&A] [answered]
I was going over https://github.com/dotnet/dotnet-docker/blob/main/samples/aspnetapp/Dockerfile.alpine-x64-slim and observed we are not creating a non root user to run the final app which seems to be a security best practice. Is this understanding correct and what needs to be done to ensure the docker uses non root user.

**Comment by mthalman** [comment, marked as answer]
This is identified in https://github.com/dotnet/dotnet-docker/issues/1772.  That would address overall documentation and the sample Dockerfiles themselves.

**Reply by rvramesh** [reply]
Thanks you so much
